contentgrid-spring-data-support/src/main/java/com/contentgrid/data/support/auditing/v1/AuditorAwareImpl.java

```diff
@@ -0,0 +1,22 @@
+package com.contentgrid.data.support.auditing.v1;
+
+import java.util.Optional;
+import org.springframework.data.domain.AuditorAware;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.oauth2.jwt.JwtClaimAccessor;
+
+public class AuditorAwareImpl implements AuditorAware<UserMetadata> {
```

Comment:
```suggestion
public class JwtAuditorAware implements AuditorAware<UserMetadata> {
```

This is a specific implementation for JWT claims